modules/olmv1-creating-a-service-account.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * extensions/ce/managing-ce.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="olmv1-creating-a-service-account_{context}"]
+= Creating a service account to manage cluster extensions
+
+Unlike {olmv0-first}, {olmv1} does not have permissions to install, update, and manage cluster extensions. Cluster administrators must create a service account and assign the role-based access controls (RBAC) required to install, update, and manage cluster extensions.
+
+[IMPORTANT]
+====
+include::snippets/olmv1-known-issue-service-accounts.adoc[]
+====
+
+.Prerequisites
+
+* Access to an {product-title} cluster using an account with `cluster-admin` permissions.
+
+.Procedure
+
+. Create a service account, similar to the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: <extension>-installer
+  namespace: <namespace>
+----
++
+.Example `extension-service-account.yaml` file
+[%collapsible]
```

Comment:
Food for thought: docs.redhat.com doesn't support `[%collapsible]` ( :sob: ). Worth just making these normal blocks?

Reply:
Good call. I debated this one, but in the end I wanted to keep it consistent with the rest of the Extensions docs set, where we make use of this design pattern a lot.

Reply:
Though it might make sense to drop them once d.o.c is gone.